samples/react/counter/app.fs

```diff
@@ -30,18 +25,18 @@ let update (msg:Msg) count =
   | Decrement -> count - 1
 
 // rendering views with React
-module R = Fable.Helpers.React
 open Fable.Core.JsInterop
 open Fable.Helpers.React.Props
+module R = Fable.Helpers.React
 
 let view count dispatch =
   let onClick msg =
     OnClick <| fun _ -> msg |> dispatch
 
   R.div []
-    [ R.button [ onClick Decrement ] [ unbox "-" ]
-      R.div [] [ unbox (string count) ]
-      R.button [ onClick Increment ] [ unbox "+" ] ]
+    [ R.button [ onClick Decrement ] [ R.str "-" ]
+      R.div [] [ R.str (string count) ]
```

Comment:
There's already a [`text` function for SVG](https://github.com/fable-compiler/fable-react/blob/6b611f6e196153c7db96f2335e6266fb3f9fbe72/src/react/Fable.Helpers.React.fs#L866) :/